### DISCUSSION TITLE
quasar mode add capacitor -  Error Unknown command "mode"

### DISCUSSION THREAD

**Discussion by glibgator** [quasarframework/quasar#14889] [CLI - Capacitor / Cordova modes]
When I reach [Add Capacitor Quasar Mode] (https://quasar.dev/quasar-cli-webpack/developing-capacitor-apps/preparation#2-add-capacitor-quasar-mode) step in the installation and run

`$ quasar mode add capacitor` 

it get " Error Unknown command "mode""

I'm on a Win10 machine. 

Also, the command `setx path "%path%;%ANDROID_SDK_ROOT%\tools;%ANDROID_SDK_ROOT%\platform-tools"` seems to have overwritten my previous path variables! not sure, if this was the command or another one, but somewhere in the installation process my previous powershell commands like `code .` and `atom .` to launch VSCode and Atom stopped working.

**Comment by MilosPaunovic** [comment]
Just a question, did you run a command with the character `$`? If so, you should run it without, so just `quasar mode add capacitor`.

**Reply by hardikhard28** [reply]
Are you sure you checked into capacitor src directory? i encountered the same error and this was the issue
